server/App.ts

```diff
@@ -19,7 +19,7 @@ app.use(Logger.consoleLog);
 
 app.use(express.static(path.resolve(__dirname, '../frontend/html/public')));
 
-app.use('/users', userRoute);
+app.use('/user', userRoute);
```

Comment:
Here is the problem. If we leave it as plural then according to specifications we have to have id passed to access specific user... ex. /users/123/inventory. However, since each user can only access their own profile, we don't use internal id of the user for identification (security reason). So, that's the reason why we have user instead of users.

Reply:
I see, thanks for clarifying